profile/README.md

```diff
@@ -8,10 +8,10 @@
 
 [link-discord]: https://discord.com/invite/0xpolygondevs
 [link-twitter]: https://twitter.com/0xPolygonDevs
-[link-website]: https://0xpolygonmiden.github.io/miden-base/introduction.html
+[link-website]: https://docs.polygon.technology/miden/
```

Comment:
This is fine for now, but I'm wondering if in the future we should link to `testnet.miden.io` and make that a reacher landing page.